ui/core/utils.ts

```diff
@@ -245,6 +245,10 @@ export function randomString(len?: number): string {
 	return str;
 }
 
+export function generateSimId() {
+	return Date.now()%1000000 + randomString(8)
```

Comment:
I added `uuid` so I think we can just use it here as well tbh. (and remove the custom UUID utils we have somewhere else for the sims:
```import { randomUUID } from '../../utils';```

Reply:
Haha yea, but it would be nice if we just use 1 function to generate all of the UUIDs and not have them spread across different files/classes. 
But if it's a placeholder feel free to delete